src/store/write_cache.rs

```diff
@@ -54,6 +62,14 @@ impl<'a, S: Store> Write for WriteCache<'a, S> {
     }
 }
 
+
+//TODO: temporary?
+impl RootHash for MapStore {
```

Comment:
Later we can make a special store for testing that implements this, since this is a dummy non-functional implementation.